### PR TITLE
updating ingress from v1beta1

### PR DESCRIPTION
**Pull request by vivian-rook**
https://phabricator.wikimedia.org/T294342

### CODE REVIEW

**Review by david-caro — APPROVED** [review]
LGTM!

File: paws/templates/public.yaml [hunk 1]
```diff
@@ -120,6 +120,9 @@ spec:
     http:
       paths:
       - backend:
-          serviceName: paws-public
-          servicePort: 8000
+          service:
+            name: paws-public
+            port:
+             number: 8000
+        pathType: ImplementationSpecific
```
Comment (supertassu) [inline paws/templates/public.yaml:127]: Why is this using `ImplementationSpecific` and not `Prefix`? Also looks like the `path` attribute is missing

**Review by supertassu — COMMENTED** [review]
one question, otherwise lgtm

File: paws/templates/public.yaml (hunk 1 above)
Comment (vivian-rook) [reply paws/templates/public.yaml:127]: Mostly the lack of path attribute in the existing config led me to leave it as the default `ImplementationSpecific`. As trying to set it as Prefix would require a path, but I wasn't sure what that path would be.

Comment (supertassu) [reply paws/templates/public.yaml:127]: I'm guessing we could use just `/` to match everything.

Comment (vivian-rook) [reply paws/templates/public.yaml]: Loads in minikube. Seems good!

**Review by supertassu — APPROVED** [review]
Didn't test but LGTM.

**Pull request merged**